frontend/src/components/FavoriteButton/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { Button } from '@carbon/react';
+import * as Icons from '@carbon/icons-react';
+import './style.scss'; // Import the styles
+
+interface FavoriteButtonProps {
+  tooltipPosition: string;
+  kind: string;
+  size: string;
+  fill: string;
+}
+
+const FavoriteButton: React.FC<FavoriteButtonProps> = ({
```

Comment:
issue: typo. Favourite.

Reply:
<img width="210" alt="image" src="https://github.com/bcgov/nr-silva/assets/39718912/c5dcdeea-c58b-4311-9bde-e5d967716533">
@RMCampos even I was confused with the spelling lol, but seems like the Carbon react goes with the UK english so I just named it that way to avoid confusion between the icon name and the component name